.github/workflows/docs.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.7]
+        python-version: ["3.8"]
 
     timeout-minutes: 30
 
```

Comment:
Shall we change:
```
- uses: actions/checkout@master
- uses: actions/setup-python@master
```

to `main` as well?

Reply:
Yes sure, thank you for spotting it